core/frontend/src/tile/TileTreeReference.ts

```diff
@@ -200,4 +200,13 @@ export abstract class TileTreeReference /* implements RenderMemory.Consumer */ {
 
   /** Return whether the geometry exposed by this tile tree reference should cast shadows on other geometry. */
   public abstract get castsShadows(): boolean;
+
+  /** Return whether this is a primary (BIM) tile tree  */
+  public get isPrimary(): boolean { return false; }

```

Comment:
You don't appear to override nor call this anywhere - did I overlook it?

Reply:
Correct - I'll remove this.

Reply:
Done.